frontend/templates/product-list/server.tsx

```diff
@@ -100,7 +113,10 @@ export const getProductListServerSideProps = ({
          }
          case ProductListType.AllTools: {
             productList = await timeAsync('findProductList', () =>
-               findProductList({ handle: { eq: 'Tools' } }, ifixitOrigin)
+               ProductListCache.get(
+                  { filters: { handle: { eq: 'Tools' } }, ifixitOrigin },
+                  hasDisableCacheGets(context)
```

Comment:
I wonder if we should change `swr-cache` to take in `context` instead of just a bool so we don't have to remember to repeat this function call in every callsite. Thoughts?

Reply:
Imo it would be better to pass an `options` object that includes flags specific to the model instead. This would help maintain a separation of concerns, as the model shouldn't be aware of the HTTP request context. By keeping this distinction, we can adhere to the MVC, where the model remains independent from the controller (Next.js request). This approach ensures that the model can be used without an HTTP context if needed.

Reply:
I implemented the options idea, let me know what you think https://github.com/iFixit/react-commerce/pull/1613/commits/5b873869a9040430f21a8b6a4ad86730778297a8

Reply:
@masonmcelvain LGTM